lhotse/testing/fixtures.py

```diff
@@ -124,3 +130,13 @@ def _with_alignment(
             for i, sub in enumerate(subwords)
         ]
         return {"subword": alignment}
+
+    def _with_custom_temporal_array(self, cut: MonoCut, frame_shift: Seconds) -> None:
+        d = TemporaryDirectory()
+        self.dirs.append(d)
+        num_frames = seconds_to_frames(cut.duration, frame_shift=frame_shift)
+        array = np.random.randint(256, size=(num_frames,))
+        with NumpyHdf5Writer(d.name) as storage:
+            cut.codebook_indices = storage.store_array(
+                key="ali1", value=array, frame_shift=frame_shift, temporal_dim=0
+            )
```

Comment:
This snippet shows how "codebook_indices" are stored and attached to a Cut manifest.